src/pages/AdminDashboard/CreateRoomForm/formValidation.tsx

```diff
@@ -1,18 +1,24 @@
-import { format } from 'date-fns'
+import { isDate } from 'date-fns'
 import i18n from '../../../i18n/i18n'
-
 import * as Yup from 'yup'
 
+const formatDate = (date: Date): Date => new Date(date.setSeconds(0, 0))
+
 export const validationSchema = Yup.object().shape({
     name: Yup.string().required(i18n.t('required-field')),
     hostName: Yup.string().required(i18n.t('required-field')),
     startAt: Yup.date()
-        .test('startAt', i18n.t('invalid-date'), (value) => {
-            //use custom format instead timestamp and isPast (date-fns). It could be have small difference between datepicker date (seconds) and now
-            const dateFormat = 'yyyy-mm-ddhh:mm'
-            const isValid =
-                format(value, dateFormat) >= format(new Date(), dateFormat)
-            return isValid
+        .nullable()
+        .transform((_, originalValue) => {
+            if (!originalValue) return null
+            const date = !isDate(originalValue)
+                ? new Date(originalValue)
+                : originalValue
+
+            return formatDate(date)
         })
-        .nullable(),
+        //min yup function doesn't work as expected. Use test instead
+        .test('startAt', i18n.t('invalid-min-date'), (value: Date) => {
+            return !value ? true : value >= formatDate(new Date())
+        }),
```

Comment:
is clicking on `Now` (`startAt` = null) taken into account by this validation logic ?

Reply:
Yes, you have a transform process before which return null if no value is provided